src/lib.rs

```diff
@@ -291,6 +291,11 @@ impl Database {
             .expect("Error getting current directory")
             .join("prelude/src");
 
+        if !prelude_path.exists() {
+            eprintln!("Warning: Prelude path does not exist: {:?}", prelude_path);
```

Comment:
This should be a panic, I think.

(and the whole function should be behind a `#[cfg(test)]` - it depends on local files)